File: TaskManager/labFinal_164/model/Comment.java
package labFinal_164.model;

import java.time.LocalDateTime;

public class Comment {
    private String id;
    private String content;
    private String Author;
    private LocalDateTime createdAt;

    public Comment(String id, String content, String Author) {
        this.id = id;
        this.content = content;
        this.Author = Author;
        this.createdAt = LocalDateTime.now();

    }


    public String getAuthor() {
        return Author;
    }
}

File: LabFinal/LabFinalTask/Model/Comment.cs
namespace LabFinalTask.Model;

public class Comment
{
    public int id;
    public string content;
    public User createdBy;
    public DateTime createdDate;
    
    public Comment(string content, User createdBy)
    {
        this.content = content;
        this.createdBy = createdBy;
        this.createdDate = DateTime.Now;
    }
}